src/main/webapp/app/exam/manage/exercise-groups/exercise-group.service.ts

```diff
@@ -13,6 +13,7 @@ export class ExerciseGroupService {
 
     constructor(private router: Router, private http: HttpClient) {}
 
+    /* istanbul ignore next */
```

Comment:
I dont’t think it’s worth artificially increasing the coverage by excluding those seven (four here, three in the other file) statements. I think it’s acceptable if they are just not tested in this case.

Reply:
🤷 Without them, they create a big fat red line in the coverage report ...
I've removed them anyways